index.bs

```diff
@@ -2425,7 +2425,7 @@ Registering Formal Objections</h3>
 	and propose changes that would remove the [=Formal Objection=];
 	these proposals <em class="rfc2119">may</em> be vague or incomplete.
 	[=Formal Objections=] that do not provide substantive arguments
-	or rationale are unlikely to receive serious consideration.
+	or rationale is unlikely to receive serious consideration.
```

Comment:
The subject is plural (`[=Formal Objections=]`), and the preceding verb (`do not`) is plural; this verb (`are`) should also be plural.
```suggestion
	or rationale are unlikely to receive serious consideration.
```